public/graph.js

```diff
@@ -420,6 +426,25 @@ class GraphController {
             d3.selectAll(".redrawElementReceived").remove();
             d3.selectAll("#receivedStack10min").remove();
             d3.selectAll("#receivedStack").remove();
+            d3.selectAll(".grid").remove();
+
+            // Add the X gridlines
+            total_received_sms_graph.append("g")			
```

Comment:
Lets split this in a separate pr, please.

Reply:
Done